backend/src/API/coffeeChatAPI.ts

```diff
@@ -145,61 +179,135 @@ export const checkMemberMeetsCategory = async (
   const haveNoCommonSubteams = (member1: IdolMember, member2: IdolMember): boolean =>
     member2.subteams.every((team) => !member1.subteams.includes(team)) &&
     member1.subteams.every((team) => !member2.subteams.includes(team));
+  let status: MemberMeetsCategoryStatus = 'no data';
```

Comment:
I know we're trying to ship this out fast, but I think it would help us bash bugs significantly if we wrote a couple unit tests for some of the cases outlined here. Just enough to convince ourselves that we handled some of the more complicated cases (i.e. PM (not on your team)) properly

Reply:
Agree! I'll write unit tests for the more complicated cases.